app/posts/index.njk

```diff
@@ -0,0 +1,44 @@
+---
+permalink: posts.xml
```

Comment:
```suggestion
permalink: feed.xml
```
I recommend using `feed.xml` or `atom.xml` as most RSS readers/browser extensions will check for that file if one isn't mentioned in the head.